Lib/test/libregrtest/run_workers.py

```diff
@@ -79,8 +79,13 @@ class MultiprocessResult:
     err_msg: str | None = None
 
 
+# Indicates that a worker thread has exited
+class WorkerExited:
+    pass
```

Comment:
(Optional micro-nit)

```suggestion
class WorkerExited:
    """Indicates that a worker thread has exited"""
```